README.md

```diff
@@ -82,7 +82,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to create client, %v", err)
 	}
-	log.Println(c.StartReceiver(context.Background(), receive));
+	if err := c.StartReceiver(context.Background(), receive)); err != nil {...};
```

Comment:
Sorry, the suggestion wasn't meant literally but as an idea. Can you please make this code compile?

Reply:
Sorry I didn't notice that the `err` is redeclared. I have fixed it in this commit https://github.com/cloudevents/sdk-go/pull/1005/commits/4086bef94236719c894064ac382f33406612ab2d